app/Models/Context.php

```diff
@@ -292,9 +292,10 @@ public static function _get($get) {
 			self::$get_unread = $tag->nbUnread();
 			break;
 		case 'T':
+			$tagDAO = FreshRSS_Factory::createTagDao();
 			self::$current_get['tags'] = true;
 			self::$name = _t('index.menu.tags');
-			self::$get_unread = 0;
+			self::$get_unread = $tagDAO->countNotRead();
```

Comment:
Is this additional request really needed?
(It is a bit expensive: ~120ms on my test system)

Reply:
If you don't do that, the filter does nothing when you select "My labels"

Reply:
I have only quickly tested but could not see a problem. Could you please post screenshots with/without ?

Reply:
The problem arise when you select "Adjust showing" in the reading configuration page.
The code that messes things up is this one:
https://github.com/FreshRSS/FreshRSS/blob/fd803875410cf056242a786260fb8ed62a733c06/app/Controllers/indexController.php#L199-L204

Reply:
The steps to reproduce is to select "Adjust showing", to add labels on multiple articles (both read and unread) and to select "My labels".
It works fine while selecting individual labels though!

Reply:
If it's not clear enough, I'll add some screenshots.

Reply:
There might be a better way to address that issue. I haven't dig into that.

Reply:
Yes please for a screenshot, as I cannot reproduce the problem when keeping the original value of zero there:

![image](https://user-images.githubusercontent.com/1008324/109306387-65230a00-783f-11eb-9583-d67fa307b49b.png)

Reply:
![Screenshot_2021-02-26 My labels · FreshRSS](https://user-images.githubusercontent.com/3056148/109355727-a7534800-784d-11eb-85ca-5b88d42aa199.png)
Here I can see 1 unread article and 2 read articles. I am expecting to see only 1 unread article when using the "Adjust showing" option.
This screenshot has been taken from the master branch.

Reply:
Ah OK, the difference is visible with the option "Show all articles in favourites by default (Applies also on labels)" is disabled.
When this option is enabled, it could stay to zero.  Let's leave it like that.